src/applications/check-in/day-of/pages/Confirmation/index.jsx

```diff
@@ -21,6 +40,37 @@ const Confirmation = () => {
     selectConfirmationData,
   );
 
+  useEffect(
+    () => {
+      if (
+        !isDayOfDemographicsFlagsEnabled ||
+        demographicsFlagsSent ||
+        getDemographicsConfirmed(window)
+      )
+        return;
+      api.v2
+        .patchDayOfDemographicsData(demographicsData)
+        .then(resp => {
+          if (resp.data.error || resp.data.errors) {
+            throw new Error();
```

Comment:
I believe this should captured since we're using the makeApiCallWithSetry util in the V2 api file. 

@mdewey do you know if there is anything else we need to do to log Api errors to sentry other that sentry util? 

Here is where I'm using it: 
https://github.com/department-of-veterans-affairs/vets-website/pull/20339/files#diff-833459979b13ebba1e56c2bdca07024a3178512dedef1ac29fcd0927285429feR156

Reply:
Confirmed that the makeAPICallWithSentry helper we are using around the V2 calls should catch this.